package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "typeorm",
   "private": true,
-  "version": "0.0.9",
+  "version": "0.1.0-alpha.1",
```

Comment:
in simple words.
let's say we have version `1.2.3` where:
`x.x.3` - is version for bug fixing or patching which does not affect public API;
`x.2.x` - is version where can be complex changes inside methods for public API, there may be marks that methods will be deprecated, but in that release they are still exists and there exists backward compatibility to deprecated methods;
`1.x.x` - is version which breaks backward compatibility, removes all methods which marked as deprecated add new big changes;

so even simple
```
 1.2.3
 ^ ^ ^
 ^ ^ ^----- PATCH
 ^ ^------- MINOR
 ^--------- MAJOR
```


So yes adding MongoDB this is big feature, public API was changed it may be major version in terms of semver